pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -223,6 +273,19 @@ rec {
     };
   };
 
+  bndtools = mirrorEclipsePlugins rec {
+    pname = "bndtools";
+    version = "6.1.0.REL-202111221555-g6b7087b";
+
+    repository = "https://bndtools.jfrog.io/bndtools/update-latest";
+    installableUnits = [
+      { id = "bndtools.main.feature.feature.group"; version = version; }
+      { id = "bndtools.m2e.feature.feature.group"; version = version; }
+      { id = "bndtools.pde.feature.feature.group"; version = version; }
+    ];
+    hash = "sha256:163l714zgx2r5yn19m8jsc9jb9rkhq6215n0ml52fiymkkxcfc8k";
+  };
+
```

Comment:
This is added as an example of using the new system as intended, and also as a sanity check since it was the original motivating example which didn't work with the previous system.